### DISCUSSION TITLE
Volume control

### DISCUSSION THREAD

**Discussion by marc22polo** [HemantKArya/BloomeeTunes#58] [General]
Hello, I searched the current discussions and issues, but cannot find anyone talking about it yet; Is there an option to control the apps volume so it isn't always on max volume?

**Comment by HemantKArya** [comment]
Hi @marc22polo 👋, there is an volume slider you can use it by sliding over cover art (on main player) up and down.

**Reply by marc22polo** [reply]
Ooooooooh, thanks, I totally missed that. Maybe it would be a good idea to write that somewhere as to not cause confusion ☺️

**Comment by HemantKArya** [comment]
yes, I guess, should add an screenshot on readme file.
